tests/integration/100.command.dev.test.js

```diff
@@ -252,6 +253,69 @@ test('Serves an Edge Function with a rewrite', async (t) => {
   })
 })
 
+test('Serves an Edge Function that includes context with site information', async (t) => {
+  await withSiteBuilder('site-with-fully-qualified-redirect-rule', async (builder) => {
```

Comment:
Not a big deal, but you could change this name to something a bit more aligned with your test.

Reply:
Do you mean specifying what type of information it returns? 🤔

Reply:
Wait, do you mean the test name or the name passed to `withSiteBuilder`? 🤔